src/Property.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Created by vasantia on 6/14/16.
+ */
+public class Property {
+
+    private String name;
+    private double price;
+    private double mortgage_Value;
+    private String color;
+    private boolean owned;
+
+    public Property(String name, Double price, Double mortgage_Value, String color){
+        this.name = name;
+        this.price = price;
+        this.mortgage_Value = mortgage_Value;
```

Comment:
Watch out for naming conventions.  All variables in java should be camelCased.  This should be `mortgageValue` instead of `mortgage_Value`